app/views/books/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= @book.title %>
+
+Average Rating: <%= @book.average_rating  %></br>
+
+ Highest Rating: <%= @book.highest_rating %></br>
+ User: <%= @book.highest_rating_review.user %>
```

Comment:
Close, but this will return a user object. Want that user's name.

```
@book.highest_rating_review.user.name
```

Given how many things are tied together there I consider moving back to the model.

Reply:
Looking more, it seems this would return a user's name, but only because of how you set up your DB. Next time use the users table instead of storing a user name on review.